src/plone/restapi/tests/test_services_querystring.py

```diff
@@ -99,10 +99,26 @@ def test_endpoint_inlines_vocabularies(self):
             "published": {"title": "Published with accent \xe9 [published]"},
             "visible": {"title": "Public draft [visible]"},
         }
+
+        self.assertTrue("external" in (idx["values"].keys()))
         self.assertTrue(
-            all(elem in idx["values"].items() for elem in expected_vocab_values.items())
```

Comment:
@ksuess I removed this check as it only was breaking with the new inclusion of tokens inside of vocabularies returned in https://github.com/plone/plone.app.querystring/pull/107. 

Also I was not able to determine what is tested here anyways in a glimps, as this nesting of loops, although short and elegant, is not really easy to read.

Reply:
I am OK with your refactoring, especially because this test 'test_endpoint_inlines_vocabularies' is not about testing the existence of review states, but about testing the format of the response of the 'querystring' endpoint.